styleguide/_themes/derek/scss/patterns/grid/ias.scss

```diff
@@ -0,0 +1,104 @@
+.ias-container {
+  @include container-fixed;
```

Comment:
If you're going to use the container-fixed mixin, you'll need to set everything for the various viewports to emulate bootstrap behavior.  This will eventually get combined into a single mixin, but for now add that. You can see how john did it here: 

https://github.com/rackerlabs/zoolander/pull/173/files#diff-df6cfdc6c265be1c82cdd82efd2ac4d8R18